node/src/chain_spec.rs

```diff
@@ -186,7 +186,7 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
     ChainSpec::from_json_bytes(&include_bytes!("../chain-specs/zkverify_testnet.json")[..])
 }
 
-#[allow(dead_code)]
+// To be used when building new testnet chain-spec
```

Comment:
Please use `///` to make this comment also a function documentation

Reply:
Modified accordingly.